src/Driver/Driver.jl

```diff
@@ -179,6 +204,11 @@ function init(; disable_gpu = false)
             parsed_args["monitor-courant-numbers"]
         Settings.monitor_courant_interval =
             parsed_args["monitor-courant-interval"]
+        Settings.checkpoint_walltime = parsed_args["checkpoint-walltime"]
+        Settings.checkpoint_keep_one = !parsed_args["checkpoint-keep-all"]
+        Settings.checkpoint_at_end = parsed_args["checkpoint-at-end"]
+        Settings.checkpoint_dir = parsed_args["checkpoint-dir"]
+        Settings.restart_from = parsed_args["restart-from"]
```

Comment:
It might make sense to call this `restart_from_nstep` or something. This way users don't need to look at the docs to see whether it's the step number or time.